packages/core/src/endowmentsToolkit.js

```diff
@@ -26,40 +26,35 @@ module.exports = endowmentsToolkit
 // Exports for testing
 module.exports._test = { instrumentDynamicValueAtPath }
 
-/**
- * @param {object} opts
- * @param {DefaultWrapperFn} [opts.createFunctionWrapper]
- * @param {boolean} [opts.handleGlobalWrite]
- * @param {Set<string>} [opts.knownWritableFields] - List of globals that can be
- *   mutated later
- */
```

Comment:
Why did this get removed?

Reply:
good catch. looks like a conflict resolution artifact. we moved the types back and forth in this branch and rebasing on main was rough.